### ISSUE TITLE
No barriers for fastFT

### ISSUE THREAD

**Issue by matthew-andrews** [Financial-Times/next-barriers#32]
e.g. https://next.ft.com/4fcc4c5d-664b-3eb4-9111-af35513c9f1d

i suspect this might some close collaboration with membership…… i'm not even sure it's clear _what_ the business logic for fast FT barriers _should_ be.

One thing I feel very, very, very strongly about:-

**The decision about whether to show fast FT articles to users should be handled by the access service.  We should provide the UUID to them and they should figure out what to do.  From Next barriers point of view Fast FT articles should be no different to any other type of articles.  If membership ask us to provide hacky-sounding date-based identifiers then alarm bells should ring and we should stop and have a meeting.**

cc @commuterjoy

// thanks to @wheresrhys

**Comment by wheresrhys** [comment]
When working on responsive fastft last year it was the same as for methode articles: first click free, and a fast article is counted content same as any other article

**Comment by matthew-andrews** [comment]
@wheresrhys that's not true.  there's a special fast ft uuid with a date in it that means once you've been granted access you can have unlimited fast ft content between 00:00 and 23.59.
